File: Iniciante/URI 1012.cs
using System;

class URI
{

  public static void Main(string[] args)
  {

    double A, B, C, pi;
    double tri, cir, tra, qua, ret;

    string line1 = Console.ReadLine();

    string[] values = line1.Split(' ');
    A = Convert.ToDouble(values[0]);
    B = Convert.ToDouble(values[1]);
    C = Convert.ToDouble(values[2]);
    pi = 3.14159;

    tri = (A * C) / 2;
    cir = pi * Math.Pow(C, 2);
    tra = ((A + B) * C) / 2;
    qua = B * B;
    ret = A * B;

    Console.WriteLine("TRIANGULO: {0:F3}", tri);
    Console.WriteLine("CIRCULO: {0:F3}", cir);
    Console.WriteLine("TRAPEZIO: {0:F3}", tra);
    Console.WriteLine("QUADRADO: {0:F3}", qua);
    Console.WriteLine("RETANGULO: {0:F3}", ret);
  }
}

File: Iniciante/URI 1012.c
#include <stdio.h>

#include <math.h>

int main()
{

  double A, B, C, pi;
  double tri, cir, tra, qua, ret;

  scanf("%lf", &A);
  scanf("%lf", &B);
  scanf("%lf", &C);
  pi = 3.14159;

  tri = (A * C) / 2;
  cir = pi * pow(C, 2);
  tra = ((A + B) * C) / 2;
  qua = B * B;
  ret = A * B;

  printf("TRIANGULO: %.3lf\n", tri);
  printf("CIRCULO: %.3lf\n", cir);
  printf("TRAPEZIO: %.3lf\n", tra);
  printf("QUADRADO: %.3lf\n", qua);
  printf("RETANGULO: %.3lf", ret);
  printf("\n");

  return 0;
}